config/initializers/health_checks.rb

```diff
@@ -0,0 +1,41 @@
+class DelayedJobNeverRanCheck < OkComputer::Check
+  attr_reader :threshold
+
+  def initialize(threshold)
+    @threshold = threshold
+  end
+
+  def check
+    never_ran = Delayed::Job.where('run_at <= ?', 5.minutes.ago).count
```

Comment:
That question IMO boils down to what exactly we want to monitor for. If we want to check whether the workers have died, then 

```
Delayed::Job.where('run_at <= ?', Delayed::Worker.sleep_delay.seconds + Delayed::Worker.max_run_time).count == Delayed::Job.count
```

would be correct and we wouldn't need the threshold. However, the alarm might be triggered a little late as `Delayed::Worker.max_run_time` is currently set to 4 hours. We might want to lower that (although I expect copying a large project might actually take some time).

If we want to check for congestion, your approach of having a more or less arbitrary value is probably correct. But even then, it would be good to have some less contingency in that value (e.g. by knowing how long a job is supposed to take, but we would have to keep track of that to get to that point)

If we also want to check whether a job is misbehaving, we will have to take the `attempts` column into account. I seem to remember @machisuji talking about one job leading to a congestion in the queue but I don't have the details on that.